src/main/scala/dpla/ingestion3/harvesters/Harvester.scala

```diff
@@ -52,6 +53,7 @@ abstract class Harvester(shortName: String,
   def harvest = {
 
     // If the output directory already exists then delete it and its contents.
+    // TODO Does this need to work with s3?
```

Comment:
I think we should stop automagically deleting our data. Particularly from S3. If you want to do it locally to be able to iterate quickly while developing, it's probably better done with a shell script that does it and then invokes the code you want to run again.